### DISCUSSION TITLE
Laravel migrate:fresh problem

### DISCUSSION THREAD

**Discussion by ghost** [yajra/laravel-oci8#653] [Q&A]
Hi,

Maybe you can help me, when I run the command "php artists migrants:cheeky" I get the following error message:

PS E:\development\01_Development\tisy> php artisan migrate:fresh

   Illuminate\Database\QueryException

  Error Code    : 4098
Error Message : ORA-04098: Trigger 'SYS.DELETE_ENTRIES' ist ungÃ¼ltig und konnte nicht neu bestÃ¤tigt werden
ORA-06512: in Zeile 3
ORA-06512: in Zeile 3
Position      : 0
Statement     : BEGIN
            FOR c IN (SELECT table_name FROM user_tables) LOOP
            EXECUTE IMMEDIATE ('DROP TABLE "' || c.table_name || '" CASCADE CONSTRAINTS');
            END LOOP;

            FOR s IN (SELECT sequence_name FROM user_sequences) LOOP
            EXECUTE IMMEDIATE ('DROP SEQUENCE ' || s.sequence_name);
            END LOOP;

            END;
Bindings      : []
 (SQL: BEGIN
            FOR c IN (SELECT table_name FROM user_tables) LOOP
            EXECUTE IMMEDIATE ('DROP TABLE "' || c.table_name || '" CASCADE CONSTRAINTS');
            END LOOP;

            FOR s IN (SELECT sequence_name FROM user_sequences) LOOP
            EXECUTE IMMEDIATE ('DROP SEQUENCE ' || s.sequence_name);
            END LOOP;

            END;)

  at E:\development\01_Development\tisy\vendor\laravel\framework\src\Illuminate\Database\Connection.php:678
    674â–•         // If an exception occurs when attempting to run a query, we'll format the error
    675â–•         // message to include the bindings with SQL, which will make this exception a
    676â–•         // lot more helpful to the developer instead of just the database's errors.
    677â–•         catch (Exception $e) {
  âžœ 678â–•             throw new QueryException(
    679â–•                 $query, $this->prepareBindings($bindings), $e
    680â–•             );
    681â–•         }
    682â–•

  1   E:\development\01_Development\tisy\vendor\yajra\laravel-pdo-via-oci8\src\Pdo\Oci8\Statement.php:712
      Yajra\Pdo\Oci8\Exceptions\Oci8Exception::("Error Code    : 4098
Error Message : ORA-04098: Trigger 'SYS.DELETE_ENTRIES' ist ungÃ¼ltig und konnte nicht neu bestÃ¤tigt werden
ORA-06512: in Zeile 3
ORA-06512: in Zeile 3
Position      : 0
Statement     : BEGIN
            FOR c IN (SELECT table_name FROM user_tables) LOOP
            EXECUTE IMMEDIATE ('DROP TABLE "' || c.table_name || '" CASCADE CONSTRAINTS');
            END LOOP;

            FOR s IN (SELECT sequence_name FROM user_sequences) LOOP
            EXECUTE IMMEDIATE ('DROP SEQUENCE ' || s.sequence_name);
            END LOOP;

            END;
Bindings      : []
")

  2   E:\development\01_Development\tisy\vendor\laravel\framework\src\Illuminate\Database\Connection.php:471
      Yajra\Pdo\Oci8\Statement::execute()
PS E:\development\01_Development\tisy>

**Comment by yajra** [comment]
I haven't encountered this issue yet but this might be related to your DB permissions? Are you using an existing schema or a new one? 

If you are using an existing schema, please be aware that fresh command will drop all you DB objects. 

If you are using a new schema and is local/dev instance: Try using this:

```sql
grant all privileges to your_schema_name identified by your_schema_password;
```
